app/views/articles/index.html.erb

```diff
@@ -3,7 +3,17 @@
 <ul>
   <% @articles.each do |article| %>
     <li>
-      <%= article.title %>
+      <%#= <a href="/articles/<%= article.id %>
```

Comment:
Do you need these comments?

Reply:
Removed unneeded comments in the index view.